addons/project/models/project.py

```diff
@@ -902,6 +902,7 @@ def get_panel_data(self):
             'user': self._get_user_values(),
             'buttons': sorted(self._get_stat_buttons(), key=lambda k: k['sequence']),
             'currency_id': self.currency_id.id,
+            'has_analytic_account_group': self.user_has_groups('analytic.group_analytic_accounting')
```

Comment:
Why is it really needed? We could display the helper when no data is displayed in project profitability? I don't really understand why we need that in JS to know if we display or not the helper. 🤔